src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -77,15 +82,31 @@ export const ChartEnvironmentSelector = ({
                 components={{
                     IndicatorSeparator: null,
                     DropdownIndicator,
-                    Option: selectOption,
-                    GroupHeading
+                    SingleValue: singleOption,
+                    GroupHeading,
                 }}
                 classNamePrefix="values-environment-select"
                 placeholder="Select Environment"
                 value={selectedEnvironment}
-                styles={getCommonSelectStyle()}
+                styles={{
```

Comment:
can we extract out styles in another component

Reply:
done